From c06d4ef19afca68537cc233004edde39309c6dc3 Mon Sep 17 00:00:00 2001
From: Elliott Sales de Andrade <quantum.analyst@gmail.com>
Date: Fri, 18 Sep 2026 02:41:03 -0400
Subject: [PATCH 1/3] Drop the extra licenses

We don't bundle c-blosc, so its license is not required.

Signed-off-by: Elliott Sales de Andrade <quantum.analyst@gmail.com>
---
 pyproject.toml | 2 --
 1 file changed, 2 deletions(-)

diff --git a/pyproject.toml b/pyproject.toml
index 185ecc3..192df28 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -36,8 +36,6 @@ maintainers = [
 license = "MIT"
 license-files = [
     "LICENSE.txt",
-    "c-blosc/LICENSE.txt",
-    "c-blosc/LICENSES/*",
 ]
 
 [project.urls]
-- 
2.56.0

